lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -1687,13 +1716,18 @@ def main():
         if not module.params.get('image'):
             module.fail_json(msg='image parameter is required for new instance')
 
-        if module.params.get('exact_count') is None:
+        if module.params.get('exact_count') is None and \
+           module.params.get('min_count') is None and \
+           module.params.get('max_count') is None:
             (instance_dict_array, new_instance_ids, changed) = create_instances(module, ec2, vpc)
         else:
             (tagged_instances, instance_dict_array, new_instance_ids, changed) = enforce_count(module, ec2, vpc)
 
     module.exit_json(changed=changed, instance_ids=new_instance_ids, instances=instance_dict_array, tagged_instances=tagged_instances)
 
+# import module snippets
+from ansible.module_utils.basic import *
```

Comment:
Can you remove these and only import specifically what is needed with the other imports?